client/src/c_console.cpp

```diff
@@ -1331,8 +1331,7 @@ int C_BasePrint(const int printlevel, const char* color_code, const std::string&
 
 		// We need to know if there were any new lines being printed
 		// in our string.
-
-		int newLineCount = std::count(logStr.begin(), logStr.end(), '\n');
+		const int newLineCount = std::count(outline, outline + strlen(outline), '\n');
```

Comment:
Looks like this line got messed up in a merge, should be `const int newLineCount = std::count(logStr.begin(), logStr.end(), '\n');`